interface/Cargo.toml

```diff
@@ -8,6 +8,10 @@ license.workspace = true
 edition.workspace = true
 
 [dependencies]
+bincode = "1.3.3"
 blake3 = "1.5.3"
 bytemuck = "1.16.0"
+hex = "0.4.3"
+parity-scale-codec = { version = "3.6.12", features = ["derive"] }
+serde = { version = "1.0.204", features = ["derive"] }
```

Comment:
Serialize/Deserialize are required for `MethodSelector` so that it can be used as a key in the symbol table BTreeMap

Reply:
Right, but I wonder why `athena_core::runtime::Program` derives `Serialize` and `Deserialize` in the first place. AFAIR, it doesn't need to. What is the use case for serializing a `Program`? IMO, it could be removed.

Reply:
Gotcha. Those have always been there, but it looks like we don't need them. I removed them. Likely due to a feature in SP1 (continuations?) that we don't have or removed.